core/README.md

```diff
@@ -103,60 +54,5 @@ If it is not supported for your platform, you will see an error if you try to st
-
-### Feature support status
-
-<details>
-<summary>Click to expand for feature support status in `wandb-core`</summary>
-
-Status legend:
-- ✅: Available: The feature is relatively stable and ready for use.
-- 🚧: In Development: The feature is available but may be unstable or incomplete.
-- ❌: Not Available: The feature is not yet available.
-
-| Category    | Feature           | Status           |
-|-------------|-------------------|------------------|
-| Experiments |                   |                  |
-|             | `init`            | ✅               |
-|             | `log`             | ✅               |
-|             | `log_artifact`    | ✅               |
-|             | `log_code`        | ✅               |
-|             | `config`          | ✅               |
-|             | `summary`         | ✅               |
-|             | `define_metric`   | ✅               |
-|             | `tags`            | ✅               |
-|             | `notes`           | ✅               |
-|             | `name`            | ✅               |
-|             | `alert`           | ✅               |
-|             | `save`            | ✅               |
-|             | `restore`         | ✅               |
-|             | `mark_preempting` | ✅               |
-|             | resume            | ✅               |
-|             | reinit            | ✅               |
-|             | Media             | ✅               |
-|             | Grouping          | ✅               |
-|             | anonymous mode    | ✅               |
-|             | offline mode      | ✅               |
-|             | disabled mode     | ✅               |
-|             | multiprocessing   | ✅               |
-|             | TensorBoard sync  | ✅               |
-|             | console logging   | ✅[^E.1]         |
-|             | system metrics    | ✅[^E.2]         |
-|             | system info       | ✅               |
-|             | auto code saving  | ✅               |
-|             | Forking           | ✅               |
-|             | Rewind            | ✅               |
-|             | Settings          | ✅               |
-| Login       |                   |                  |
-|             | default entity    | ✅               |
-|             | team entity       | ✅               |
-|             | service account   | ✅               |
-| CLI         |                   | ✅               |
-| Artifacts   |                   | ✅               |
-| Sweeps      |                   | ✅               |
-| Launch      |                   | ✅               |
-
-[^E.1]: Only raw console logging is supported.
-[^E.2]: Supported system metrics: CPU, Memory, Disk, Network, NVIDIA GPU, AMD GPU, Apple GPU, AWS Trainium.
-<details>
+To revert to the old SDK backend, simply add `wandb.require("legacy-service")` to your script or set the `WANDB__REQUIRE_LEGACY_SERVICE` environment variable as `TRUE`.
```

Comment:
do you want to add a comment about `wandb.require('core')` and `wandb.require("legacy-service")` being mutually exclusive and will raise an error if used together?

Reply:
Thats a good call. I'll add that.